src/node_file.cc

```diff
@@ -95,11 +97,74 @@ class FSReqWrap: public ReqWrap<uv_fs_t> {
   DISALLOW_COPY_AND_ASSIGN(FSReqWrap);
 };
 
+class Path {
+  public:
+    explicit Path(v8::Isolate* isolate, v8::Local<v8::Value> value);
```

Comment:
This is a `.cc` file. Should be able to use `using v8::Isolate` at the top and not need to use the namespace prefix.